tools/testbench/topology.c

```diff
@@ -256,7 +189,7 @@ int tplg_register_pga(struct tplg_context *ctx)
 int tplg_register_pipeline(struct tplg_context *ctx)
 {
 	struct sof *sof = ctx->sof;
-	struct sof_ipc_pipe_new pipeline = {0};
+	struct sof_ipc_pipe_new pipeline = {{0}};
```

Comment:
rebase needed?

Reply:
This came with Liam's testbench cleanup work, after a fix for these. Should this change for xcc be a separate commit? I was feeling lazy and included it here.

Reply:
Fair enough, up to you.

Reply:
normal way with memset(ptr, 0, size)?

Reply:
Why would you use a longer and more tedious `memset` when you let the compiler figure out everything correctly?

The rest of the code uses `{{0}}`